UI_DSM/UI_DSM.Client/Pages/NormalUser/ReviewTaskPage/ReviewTaskPage.razor.cs

```diff
@@ -206,8 +216,15 @@ private void OnSelectedModelChange()
         /// <returns>A <see cref="Task" /></returns>
         private async Task OnModelSelectorChanged()
         {
+            IReusableView interfaceView = null;
```

Comment:
Rename the variable interfaceView. Not only the interfaceView is an IReusableView